src/tactic/where.lean

```diff
@@ -4,213 +4,169 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Keeley Hoek
 -/
 
-import data.list.defs tactic.core
+import data.list.defs
+import tactic.core
 
 /-!
 # The `where` command
 
-When working in a Lean file with namespaces, parameters, and variables,
-it can be confusing to identify what the current "parser context" is.
-The command `#where` tries to identify and print information about the current location,
-including the active namespace, open namespaces, and declared variables.
+When working in a Lean file with namespaces, parameters, and variables, it can be confusing to
+identify what the current "parser context" is. The command `#where` identifies and prints
+information about the current location, including the active namespace, open namespaces, and
```

Comment:
Is this true even with the "open bug" mentioned below?

Reply:
Oh, below in the file!

Not really. Since the open namespaces don't work the way I expected (as explained by Mario in the thread), in rare circumstances it is not possible to exactly replicate the way names are resolved at the current point in a file where `#where` is typed. But what is returned by `#where` is the closest approximation to what you get, and I frankly I think this subtlety might never come up.